app/components/node-navbar/component.ts

```diff
@@ -0,0 +1,27 @@
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import defaultTo from 'ember-osf-web/utils/default-to';
+
+enum Tabs {
+    Title = 'title',
+    Files = 'files',
+    Wiki = 'wiki',
+    Analytics = 'analytics',
+    Registrations = 'registrations',
+    Contributors = 'contributors',
+    Addons = 'addons',
+    Settings = 'settings',
+}
+
+export default class NodeNavbar extends Component {
+    node: Node;
```

Comment:
This `Node` is actually [this Node](https://github.com/Microsoft/TypeScript/blob/master/lib/lib.d.ts#L13515), not `ember-osf-web/models/node`. You'll need to import it to avoid any type confusion.